dao/clickhouse/migrations/010_validators.up.sql

```diff
@@ -152,4 +152,5 @@ from (
           OR signatures > 0
           OR reg_expiration >= (select max(blk_epoch) from blocks)) prep
        ANY
-       LEFT JOIN public_validators USING reg_entity_address; --public validators 134 rows
\ No newline at end of file
+       LEFT JOIN public_validators USING reg_entity_address --public validators 134 rows
+;
```

Comment:
Need the semicolon on the next line here, or else `--public validators 134 rows` get interpreted as an empty statement.